app/src/main/res/layout/parking_dialog_fragment.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:padding="@dimen/dialog_fragment_text_padding"
+    android:background="@color/main_activity_purple_background">
+
+    <Button
+        android:id="@+id/button_ok_dialog_fragment"
+        android:layout_width="wrap_content"
+        android:layout_height="match_parent"
+        android:text="@string/text_button_ok_dialog_fragment"
+        app:layout_constraintTop_toBottomOf="@+id/editTextNumber"
+        app:layout_constraintStart_toStartOf="@+id/textView_dialog_fragment" />
```

Comment:
This should be text_view_dialog_fragment

Reply:
Done